controllers/clusters/kafka_controller_test.go

```diff
@@ -58,7 +59,7 @@ var _ = Describe("Kafka Controller", func() {
 					return false
 				}
 
-				return kafka.Status.ID == openapi.CreatedID
+				return kafka.Status.ID != ""
```

Comment:
why have you changed this?

Reply:
Because I used uuid to generate IDs for entities. Reverted